### DISCUSSION TITLE
Welcome to lava-optimization Discussions!

### DISCUSSION THREAD

**Discussion by mgkwill** [lava-nc/lava-optimization#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AjayKumar-LVLS** [comment]
Hi,
I see in the recent commit to the lava optimization there is an introduction of QUBO problem. When we can expect a complete QUBO optimization library release?
Thanks in advance

**Reply by awintel** [reply]
The QUBO solver for CPU and Loihi 2 backends will be released next week.

**Reply by AjayKumar-LVLS** [reply]
Great! Thanks for answering.

**Reply by AjayKumar-LVLS** [reply]
I would appreciate a reply on further updates to QUBO optimization library release!

**Reply by awintel** [reply]
We are asymptotically converging. We just pushed the release to tomorrow to take care of some last minute clean ups.
